.github/CODEOWNERS

```diff
@@ -18,3 +18,4 @@
 /README.md @chipkent @rcaudy @margaretkennedy
 /TRIAGE.md @chipkent
 /licenses @chipkent
+/docker @devinrsmith
```

Comment:
Since there is a non-zero chance that Docker creates a time-critical problem and a non-zero chance you will go on vacation, can we add a backup?  e.g. me, Colin, etc.